### PR TITLE
Create hiroarchiver.md

### PR DESCRIPTION
**Pull request by trichards-hiro**
## Description

Created a new Hiro Archiver file for inclusion into the Hiro docs.

1. Motivation for change? We had an existing file outside of the Hiro docs set. This includes the file in the docs.
2. What was changed? A new file was created from an existing readme file that was created.
3. Link to relevant issues?

## Checklist

- [x] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] New links to files and images were verified
- [x] For fixes/refactors: all existing references were identified and replaced
- [x] [Style guide](https://developers.google.com/style) was reviewed and applied
- [x] Clear code samples were provided
- [x] People were tagged for review

### CODE REVIEW

File: docs/hiroarchiver.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+# Hiro Archiver
+
+## What is the Hiro Archiver
+The purpose of the Hiro Archiver is to enable developers and users of the Hiro ecosystem to download and use any of the 
+published datasets needed to run a Stacks environment. This is a public service Hiro offers as a benefit to the 
+Stacks community.  
+
+## How to download
+Hiro pushes out new datasets nightly. The best way to download these datasets is by constructing a download link.<br>
```
Comment (sabbyanandan) [inline docs/hiroarchiver.md:14]: Having the `<br>` tag here and no closure for it is causing the generated static html to break. Hence, the PR build+preview action is failing.

Maybe we could remove this; the line-break is automatically rendered, anyway.

Comment (trichards-hiro) [reply docs/hiroarchiver.md]: fixed

**Comment by sabbyanandan** [comment]
Hi, @trichards-hiro. The left-nav linking is missing, right? How will this be accessible?

**Comment by LakshmiLavanyaKasturi** [comment]
> Hi, @trichards-hiro. The left-nav linking is missing, right? How will this be accessible?

+1

**Comment by trichards-hiro** [comment]
@sabbyanandan and @LakshmiLavanyaKasturi : I am working on the left nav to make this file accessible.

**Comment by sabbyanandan** [comment]
@trichards-hiro: This was merged but the left-nav link to this page is missing still, correct? Or maybe addressed in a different commit?

**Pull request merged**